BUILDING.md

```diff
@@ -0,0 +1,68 @@
+## Build the development environment
+
+The first prerequisite of properly building SwarmKit targets is to have a Go
+development environment setup. Please follow [How to Write Go Code](https://golang.org/doc/code.html)
+for proper setup. If done correctly, you should have a GOROOT and GOPATH set in the
+environment.
+
+If a Go development environment is setup, one can use `go get` to install the
+`swarmkit`:
+
+    go get github.com/docker/swarmkit/...
+
+The above will install the source repository into the `GOPATH`. Any binaries
```

Comment:
This command installs the source repository into the `GOPATH` and builds the binaries into $GOPATH/bin`. Alternatively run ` go get github.com/docker/swarmkit/... -d` to download the source.